app-sso/getting-started/provision-auth-server.hbs.md

```diff
@@ -24,209 +25,75 @@ tanzu package installed list -A | grep "sso.apps.tanzu.vmware.com"
-from which authorization server a specific client obtains tokens from.
-
-The `sso.apps.tanzu.vmware.com/allow-client-namespaces` annotation restricts the namespaces in which you can create
-`ClientRegistrations` targeting this authorization server. In this case, the authorization server only picks up
-client registrations in the `default` namespace.
-
-The `sso.apps.tanzu.vmware.com/allow-unsafe-...` annotations enable "development mode" features, useful for testing.
-Those should not be used for production-grade authorization servers.
-
-For more information about annotations and labels in `AuthServer` resource, see [Annotation and labels](../how-to-guides/service-operators/metadata.md).
-
-### TLS & issuer URI
+and you should see:
 
-```yaml
-spec:
-  tls:
-    deactivated: true
+```shell
+NAME                          STATUS
+my-unsafe-test-login-example  Ready
 ```
 
-The `tls` field configures whether and how to obtain a certificate for an `AuthServer` to secure its issuer URI.
-If you deactivate `tls`, the issuer URI uses plain HTTP.
-
-> **Caution** Plain HTTP access is for development purposes only and must never be used in production.
-For more information about the production readiness with TLS, see [Issuer URI & TLS](../how-to-guides/service-operators/issuer-uri-and-tls.md).
-
-### Token Signature
-
-```yaml
----
-apiVersion: sso.apps.tanzu.vmware.com/v1alpha1
-kind: AuthServer
-# ...
-spec:
-  tokenSignature:
-    signAndVerifyKeyRef:
-      name: "authserver-signing-key"
----
-apiVersion: secretgen.k14s.io/v1alpha1
-kind: RSAKey
-metadata:
-  name: authserver-signing-key
-  namespace: default
-spec:
-  secretTemplate:
-    type: Opaque
-    stringData:
-      key.pem: $(privateKey)
-      pub.pem: $(publicKey)
-```
+If you want to see your `AuthServer`'s URI, look at:
 
-The token signing key is the private RSA key used to sign ID Tokens,
-using [JSON Web Signatures](https://datatracker.ietf.org/doc/html/rfc7515), and clients use the public key to verify the
-provenance and integrity of the ID tokens. The public keys used for validating messages are published
-as [JSON Web Keys](https://datatracker.ietf.org/doc/html/rfc7517) at `{.status.issuerURI}/oauth2/jwks`.
-
-The `spec.tokenSignature.signAndVerifyKeyRef.name` references a secret containing PEM-encoded RSA keys, both `key.pem`
-and `pub.pem`. In this specific example, we are
-using [Secretgen-Controller](https://github.com/vmware-tanzu/carvel-secretgen-controller), a TAP dependency, to generate
-the key for us.
-
-Learn more about [Token Signatures](../how-to-guides/service-operators/configure-token-signature.hbs.md).
-
-### Identity providers
-
-```yaml
-spec:
-  identityProviders:
-    - name: "internal"
-      internalUnsafe:
-        users:
-          - username: "user"
-            password: "password"
-            email: "user@example.com"
-            roles:
-              - "user"
+```shell
+kubectl get authserver -A
```

Comment:
**[thought]:** This makes me think that extending `CUTL`'s print-columns with `Issuer URI` would be really handy.

Reply:
Created [#185664596](https://www.pivotaltracker.com/story/show/185664596)